### PR TITLE
Fix master build on travis + add tox

### PR DESCRIPTION
**Pull request by ale-rt**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -0,0 +1,14 @@
+[tox]
+envlist =
+   py27,
+   py36,
+   py37
+
+[testenv]
+commands =
+    {envbindir}/buildout -c {toxinidir}/buildout.cfg buildout:directory={envdir} buildout:develop={toxinidir} bootstrap
+    {envbindir}/buildout -c {toxinidir}/buildout.cfg buildout:directory={envdir} buildout:develop={toxinidir} install test
```
Comment (mgedmin) [inline tox.ini:10]: Does this actually run the tests?  IIRC `buildout install test` creates a `bin/test` part but doesn't run it.

Comment (ale-rt) [reply tox.ini:10]: No I run the tests manually and travis does its own thing, anyway I will fix this

**Comment by ale-rt** [comment]
Superseded by #12

File: tox.ini (hunk 1 above)
Comment (ale-rt) [reply tox.ini:10]: @mgedmin I added the test run here: https://github.com/zopefoundation/z3c.relationfield/pull/12/commits/328f26f12a885393f831bafec7d628435c10cfdf as part of #12

**Pull request closed without merging**